sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero_command_buffer.cpp

```diff
@@ -252,11 +252,12 @@ static ur_result_t getEventsFromSyncPoints(
   return UR_RESULT_SUCCESS;
 }
 
+// Shared by all memory read/write/copy PI interfaces.
 // Helper function for common code when enqueuing memory operations to a command
 // buffer.
 static ur_result_t enqueueCommandBufferMemCopyHelper(
-    ur_exp_command_buffer_handle_t hCommandBuffer, void *Dst, const void *Src,
-    size_t Size, uint32_t NumSyncPointsInWaitList,
+    ur_command_t CommandType, ur_exp_command_buffer_handle_t hCommandBuffer,
```

Comment:
```suggestion
    ur_command_t CommandType, ur_exp_command_buffer_handle_t CommandBuffer,
```
I know it was like this before but just for naming consistency.